test/suites/auth.sh

```diff
@@ -100,6 +100,8 @@ test_authorization() {
   BROWSER=curl lxc remote add --accept-certificate oidc "${LXD_ADDR}" --auth-type oidc
 
   ! lxc auth identity group add oidc/test-user@example.com not-found || false # Group not found
+  result="$(! lxc auth identity group add oidc/test-user@example.com not-found 2>&1)"
```

Comment:
Can we try a multi group error to check separator

Reply:
Done. I didn't because the command doesn't let you add more than one. Now I'm using curl to check it.